Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM joshmahowald/cloud-workstation AS baseworkstation
```

Comment:
While this image may be useful for certain workflows, it goes beyond the scope of what is needed to create a functional image containing Terraform and Kitchen-Terraform. Additional content like what is contained in this image should be layered on top by a different image, not used as the basis for our image.

Reply:
I'll remove it for confusion, but it's actually a multi stage build, and as such only the pieces which are subsequently copied in make it's way in.